sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -281,6 +281,29 @@ export class ManagementClient extends LinkEntity {
       : undefined;
   }
 
+  /**
+   * Evaluates the `associatedLinkName` to use for given operation.
```

Comment:
> Evaluates the `associatedLinkName` to use for **given operation.**

The method does not take an operation as an argument. Therefore, this description is misleading.
Please consider rephrasing. The older `Helper function to retrieve active receiver name, if it exists.` can  be used as a reference.

Reply:
Updated